spec/features/posts_spec.rb

```diff
@@ -0,0 +1,92 @@
+    FactoryBot.create(:comment, post: @user1.posts.first, author: @user3)
+    FactoryBot.create(:comment, post: @user2.posts.first, author: @user1)
+    FactoryBot.create(:comment, post: @user2.posts.first, author: @user3)
+  end
+  describe 'Posts Index' do
+    scenario 'I can see the user\'s profile picture' do
+      visit user_posts_path(@user1.id)
+      expect(page).to have_selector("img[src='https://i.pravatar.cc/300?img=1']")
+    end
+
+    scenario 'I can see the user\'s name' do
+      visit user_posts_path(@user1.id)
+      expect(page).to have_content(@user1.name)
+    end
+
+    scenario 'I can see the number of posts of the user' do
+      visit user_posts_path(@user1.id)
+      expect(page).to have_content(@user1.posts_counter)
+    end
+
+    scenario 'I can see a post\'s title' do
+      visit user_posts_path(@user1.id)
+      expect(page).to have_content(@user1.posts.first.title)
+    end
+
+    scenario 'I can see a post\'s body' do
+      visit user_posts_path(@user1.id)
+      expect(page).to have_content(@user1.posts.first.body)
+    end
+
+    scenario 'I can see the first comment on a post' do
+      visit user_posts_path(@user1.id)
+      expect(page).to have_content(@user1.posts.first.comments.first.body)
+    end
+
+    scenario 'If i click on a post, it directs me to the post\'s show page' do
+      visit user_posts_path(@user1.id)
+      click_on @user1.posts.first.title
+      expect(page).to have_current_path(user_post_path(@user1.id, @user1.posts.first.id))
+    end
+    scenario 'I can see a section for pagination if there are more posts than fit on the view' do
+      visit user_posts_path(@user2)
+      expect(page).to have_selector('div.flickr_pagination a')
+    end
+  end
+
+  describe 'Posts Show' do
+    scenario 'I can see the post title' do
+      visit user_post_path(@user1.id, @user1.posts.first.id)
+      expect(page).to have_content(@user1.posts[0].title)
+    end
+
+    scenario 'I can see who wrote the post' do
+      visit user_post_path(@user1.id, @user1.posts.first.id)
+      expect(page).to have_content(@user1.name)
+    end
+
+    scenario ' I can see how many comments the post has' do
+      visit user_post_path(@user1.id, @user1.posts.first.id)
+      expect(page).to have_content(@user1.posts.first.comments_counter)
+    end
+
+    scenario 'I can see how many likes the post has' do
+      visit user_post_path(@user1.id, @user1.posts.first.id)
+      expect(page).to have_content(@user1.posts.first.likes_counter)
+    end
+
+    scenario 'I can see the username of each commentor' do
+      visit user_post_path(@user1.id, @user1.posts.first.id)
+      expect(page).to have_content(@user2.name)
+      expect(page).to have_content(@user3.name)
+    end
+
+    scenario 'I can see the comment by each commentor' do
+      visit user_post_path(@user1.id, @user1.posts.first.id)
+      expect(page).to have_content(@user2.comments.first.body)
+    end
+  end
+end
```

Comment:
- [ ] In the posts show page tests you are missing tests that asserts we can see the post body. Kindly add that.